cl21.md

```diff
@@ -12,6 +12,24 @@ and brings some new welcome features:
 * symbols organized into several packages,
 * written in pure Common Lisp.
 
+CL21 is a bit disruptive in the sense that it redefines usual symbols
+(specially for the generic functions). You might want to check
+other related projects, that go beyond
+[Alexandria](https://common-lisp.net/project/alexandria) and stay
+"good citizens" of the CL world:
+
+- [rutils](https://github.com/vseloved/rutils/blob/master/docs/ann-rutils.md) -
+  a comprehensive and all-encompassing suite of syntactic utilities to
+  support modern day-to-day Common Lisp development. It adds readtable
+  literal syntax for shorter lambdas, hash-tables and vectors
+  (<code>#`…`</code>, `#h` and `#{…}`, `#v`), some generic functions
+  (whose name start with `generic-`),
+  [shortcuts](https://github.com/vseloved/rutils/blob/master/core/abbr.lisp)
+  for standard operators, and many helpful functions.
+- [Serapeum](https://github.com/TBRSS/serapeum) is a set of utilities
+  beyond Alexandria, as a supplement. See its
```

Comment:
might want to annotate what it really does?

Reply:
allright, done, but I can't do much more, it's a lot of functions and macros often technical.